### PR TITLE
[feature/336342: Do you hold a waste permit exemption?]

### PR DESCRIPTION
**Pull request by shehzad-ismail8**

### CODE REVIEW

File: src/EPR.Accreditation.Facade/Services/WastePermitService.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+﻿using EPR.Accreditation.Facade.Common.Dtos;
+using EPR.Accreditation.Facade.Common.Dtos.Portal;
+using EPR.Accreditation.Facade.Common.RESTservices.Interfaces;
+using EPR.Accreditation.Facade.Services.Interfaces;
+
+namespace EPR.Accreditation.Facade.Services
+{
+    public class WastePermitService : IWastePermitService
+    {
+
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Facade/Services/WastePermitService.cs]: blank line

File: src/EPR.Accreditation.Facade/Services/WastePermitService.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    WastePermitExemption = permitExemption.HasPermitExemption
+                }
+            };
+
+            await _httpAccreditationService.UpdateAccreditation(
+                accreditationExternalId,
+                accreditation
+                );
+        }
+
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Facade/Services/WastePermitService.cs]: blank line

File: src/EPR.Accreditation.Facade/Services/WastePermitService.cs (hunk 1 above)
Comment (shehzad-ismail8) [reply src/EPR.Accreditation.Facade/Services/WastePermitService.cs]: Removed

File: src/EPR.Accreditation.Facade/Services/WastePermitService.cs (hunk 2 above)
Comment (shehzad-ismail8) [reply src/EPR.Accreditation.Facade/Services/WastePermitService.cs]: Removed

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**